.github/env

```diff
@@ -1,2 +1,2 @@
 GOLANG_VERSION=1.24
-GOLANGCI_LINT_VERSION=v1.64
```

Comment:
I think if we do this we have to "backport" what we have on main branch. Like I did in https://github.com/helm/helm/pull/30752 but there is a little more after in golang ci lint config file.

Reply:
Also related https://github.com/helm/helm/pull/30771/files#r2050843093

Reply:
I believe @benoittgt is correct on this one...

Reply:
I was confused by outdated local branch. https://github.com/helm/helm/pull/30785 is the fix which needs to be backported